search/search.py

```diff
@@ -11,7 +14,29 @@ def get_search_blueprint() -> Blueprint:
 
     @blueprint.route('/search', methods=['GET'])
     def search() -> Response:
+
+        # get query string and decode bytes. If present add it to es query
+        kwargs = {}
+        query_string = request.query_string.decode('utf-8')
```

Comment:
a query string is not part of the specification yet?

Reply:
It likely wouldn't look the same as the format expected by ElasticSearch; as much as it can be useful for debugging to allow it here it violates the encapsulation of the underlying database, plus not sure of the security implications (imagine allowing to pass in SQL fragments for your queries)

Reply:
Are you asking me to remove the ability to search for things?

Reply:
Waiting for more input from other people, but it's out of the scope of https://github.com/libero/libero/issues/107 (and if the requirement was specified, it's unlikely the ElasticSearch 6.x API and the Libero API would coincide on what parameters are passed/what values they have)

Reply:
Currently no query string parameter are defined (deliberately), so needs to be removed.